scripts/bundleConfig.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const config = require('../shared/data/constants.js')
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args))
```

Comment:
OMG what happened here? Why are we importing node-fetch only after fetch is called? Can we make it a bit less interesting?

Reply:
This is the standard way to import it now when not in a mjs file: https://www.npmjs.com/package/node-fetch I changed to .mjs which still equally feels weird to me.